### PR TITLE
[63] Style the Contact Us Page using CSS

### PR DESCRIPTION
**Pull request by bilkisutace**
## Task: `[63] Style the Contact Us Page using CSS`
- `Styled the Contact Us Page using CSS`
mobile-signup
mobile-signup2
tablet-signup
tablet-signup2
web-signup
web-signup2

### CODE REVIEW

**Comment by Ireti203** [comment]
Weldone @bilkisutace 

Your social media icons looks lumped together. Can you give it some sort of padding.

File: submissions/bilkisuTace/css/contactus.css [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+}
+.nav-links a {
+    float: right;
+    font-size: 30px;
+    padding-left: 50px;
+    color: white;
+    text-decoration: none;
+}
+/***************************
+            BODY
```
Comment (Ireti203) [inline submissions/bilkisuTace/css/contactus.css:41]: Remove extra white space. You can put the comment on a single line

File: submissions/bilkisuTace/css/contactus.css [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+    float: right;
+    margin-right: 10%;
+    color: white;
+    border: 0 solid black;
+    width: 150px;
+    height: 40px;
+    font-size: 18px;
+}
+/**********************************
+            FOOTER
```
Comment (Ireti203) [inline submissions/bilkisuTace/css/contactus.css:135]: Same here

File: submissions/bilkisuTace/html-forms/contactUs.html [hunk 3]
```diff
@@ -0,0 +1,115 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>Contact Us | Ladies & Gents</title>
+
```
Comment (Ireti203) [inline submissions/bilkisuTace/html-forms/contactUs.html:7]: Remove whitespace.

File: submissions/bilkisuTace/html-forms/contactUs.html [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+	<link href="https://fonts.googleapis.com/css2?family=Montserrat&display=swap" rel="stylesheet"/>
+
```
Comment (Ireti203) [inline submissions/bilkisuTace/html-forms/contactUs.html:9]: Remove whitespace

File: submissions/bilkisuTace/html-forms/contactUs.html [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+	<link rel="stylesheet" type="text/css" href="../css/contactus.css"/>
+</head>
+<body>
+	<div class="container">
+
```
Comment (Ireti203) [inline submissions/bilkisuTace/html-forms/contactUs.html:14]: Remove whitespace.

File: submissions/bilkisuTace/html-forms/contactUs.html [hunk 6]
```diff
@@ -0,0 +1,115 @@
...
+						<p class="p-heading"><b>Or Connect with us:</b></p>
+						<ul class="body-social">
+                            <li><img src="../html/images/facebook.jpg"/></li>
+                            <li><img src="../html/images/twitter.jpg"/></li>
+                            <li><img src="../html/images/gmail.jpg"/></li>
+                            <li><img src="../html/images/youtube.jpg"/></li>
+                            </ul>
+					</span>
+			</div>
+			<div class="div-form">
```
Comment (Ireti203) [inline submissions/bilkisuTace/html-forms/contactUs.html:48]: Kindly format properly.

File: submissions/bilkisuTace/html-forms/contactUs.html [hunk 7]
```diff
@@ -0,0 +1,115 @@
...
+                        <a href="#">About Us </a><br/><br/>
+                        <a href="#">Terms and Conditions </a>
+					</span>
+				</div>
+				<div class="footer-news">
+					<div class="news">
+						<span>
+                            <form>
+                                 <label id="email" for="email"><strong>Sign up for our Newsletter</label></strong><br/><br>
+                                <input type="email" name="email" id="email" placeholder="Enter your email address"/>
```
Comment (Ireti203) [inline submissions/bilkisuTace/html-forms/contactUs.html:85]: Kindly format this file properly.

**Review by Ireti203 — CHANGES_REQUESTED** [review]

**Comment by adaugochi** [comment]
From the screenshot you sent, the form card isn't aligned properly. Kindly check the mockup again

**Comment by bilkisutace** [comment]
mobileview1
mobileview2
tabletview
webview

File: submissions/bilkisuTace/css/contactus.css [hunk 8]
```diff
@@ -0,0 +1,242 @@
...
+  .news {
+    padding-top: 5%;
+    padding-left: 0;
+  }
+}
+/*NAVIGATION*/
+.topnav {
+    width: 100%;
+    display: flex;
+    flex-direction: row;
```
Comment (adaugochi) [inline submissions/bilkisuTace/css/contactus.css:22]: by default, it has `row` as flex-direction. So there is no need to using it

File: submissions/bilkisuTace/css/contactus.css [hunk 9]
```diff
@@ -0,0 +1,242 @@
...
+    padding-top: 20px;
+    height: 10%;
+    vertical-align: middle;
+}
+.nav-logo {
+    width: 10%;
+}
+.nav-links {
+    display: flex;
+    flex-direction: row;
```
Comment (adaugochi) [inline submissions/bilkisuTace/css/contactus.css:35]: same here

File: submissions/bilkisuTace/html-forms/contactUs.html [hunk 10]
```diff
@@ -0,0 +1,103 @@
...
+				<span>
+					<img class="img" src="../html/images/call.png" alt="">
+					<p class="p-heading"><b>Please contact us via these numbers:</b></p>
+					<p>+23480808001</p>
+					<p>+23480808002</p><br/>
+					<img class="img" src="../html/images/env1.png" alt="">
+					<p class="p-heading"><b>Email Address:</b></p>
+					<p>support@ladies&gents.com</p><br>
+					<p class="p-heading"><b>Or Connect with us:</b></p>
+					<ul class="body-social">
```
Comment (adaugochi) [inline submissions/bilkisuTace/html-forms/contactUs.html:37]: kindly indent your code properly

**Review by adaugochi — CHANGES_REQUESTED** [review]

File: submissions/bilkisuTace/html-forms/contactUs.html [hunk 11]
```diff
@@ -0,0 +1,103 @@
...
+						<a href="contact.html">Contact Us </a><br/><br/>
+                        <a href="#">About Us </a><br/><br/>
+                        <a href="#">Terms and Conditions </a>
+					</span>
+				</div>
+				<div class="footer-news">
+					<div class="news">
+						<span>
+                            <form>
+                                 <label id="email" for="email"><b id="news-sign">Sign up for our Newsletter</label></b><br/><br>
```
Comment (ToluAkin) [inline submissions/bilkisuTace/html-forms/contactUs.html:75]: Remove the break on this label and style with css @bilkisutace

File: submissions/bilkisuTace/html-forms/contactUs.html [hunk 12]
```diff
@@ -0,0 +1,103 @@
...
+                                <input class="news-email" type="email" name="email" id="email" placeholder="Enter your email address"/>
+                                <button class="white-button" type="button">SUBMIT</button>
```
Comment (ToluAkin) [inline submissions/bilkisuTace/html-forms/contactUs.html:77]: I think you should reduce the font-size of your button. The button seem to be bigger than the text input. @bilkisutace

File: submissions/bilkisuTace/html-forms/contactUs.html [hunk 13]
```diff
@@ -0,0 +1,103 @@
...
+						</fieldset>
+					</form>
+				</span>
+			</div>
+		</div>
+		<div class="footer">
+			<div class="footer-body">
+				<div class="footer-links">
+					<span class="links">
+						<a href="contact.html">Contact Us </a><br/><br/>
```
Comment (ToluAkin) [inline submissions/bilkisuTace/html-forms/contactUs.html:66]: You should put the links in separate span elements or you use list to render them. @bilkisutace Also, remove the br around the links

**Review by ToluAkin — CHANGES_REQUESTED** [review]
@bilkisutace Remove all break elements in your html file

**Review by adaugochi — APPROVED** [review]

**Pull request merged**